sdk/keyvault/Microsoft.Azure.KeyVault/tests/TestFramework/KeyVaultTestFixture.cs

```diff
@@ -31,7 +31,24 @@ public class KeyVaultTestFixture : IDisposable
         public string keyVersion;
         public KeyIdentifier keyIdentifier;
         public ClientCredential clientCredential;
-        public RetryPolicy retryExecutor;
+        public RetryPolicy retryExecutor
+        {
+            get
+            {
+                switch (HttpMockServer.Mode)
+                {
+                    case HttpRecorderMode.None:
```

Comment:
IIRC, "none" is the same as "live", which in track 2 is effectively "record" mode sans updating recording files. Could these first 2 cases just fall through to the same result?

Reply:
They had different values in the original code so I just copied them.